resources/public/api/conf/4.0/application.yaml

```diff
@@ -4,6 +4,10 @@ info:
   version: "4.0"
   title: Individual Losses (MTD)
   description: |
+    The following groups are stateful in the Sandbox:
+    * Brought Forward Losses
+    * Loss Claims
```

Comment:
Should say 'The following groups of endpoints are stateful in the Sandbox:'

The groups in the bullet point should link to the relevant sections eg: https://github.com/hmrc/individuals-dividends-income-api/blob/657bac09da4954772886fd5ecc1ff534e1e0d54f/resources/public/api/conf/1.0/application.yaml#L7-L9